packages/design-tokens/docs/ColorTokens.stories.tsx

```diff
@@ -0,0 +1,261 @@
+            <ColorDemo name="700" color={theme.color.red[700]} />
+          </Stack>
+          <Stack gapSize={0.5}>
+            <Heading variant="heading-3">Orange</Heading>
+            <ColorDemo name="100" color={theme.color.orange[100]} />
+            <ColorDemo name="200" color={theme.color.orange[200]} />
+            <ColorDemo name="300" color={theme.color.orange[300]} />
+            <ColorDemo name="400" color={theme.color.orange[400]} />
+            <ColorDemo name="500" color={theme.color.orange[500]} />
+            <ColorDemo name="600" color={theme.color.orange[600]} />
+            <ColorDemo name="700" color={theme.color.orange[700]} />
+          </Stack>
+          <Stack gapSize={0.5}>
+            <Heading variant="heading-3">Green</Heading>
+            <ColorDemo name="100" color={theme.color.green[100]} />
+            <ColorDemo name="200" color={theme.color.green[200]} />
+            <ColorDemo name="300" color={theme.color.green[300]} />
+            <ColorDemo name="400" color={theme.color.green[400]} />
+            <ColorDemo name="500" color={theme.color.green[500]} />
+            <ColorDemo name="600" color={theme.color.green[600]} />
+            <ColorDemo name="700" color={theme.color.green[700]} />
+          </Stack>
+          <Stack gapSize={0.5}>
+            <Heading variant="heading-3">Yellow</Heading>
+            <ColorDemo name="100" color={theme.color.yellow[100]} />
+            <ColorDemo name="200" color={theme.color.yellow[200]} />
+            <ColorDemo name="300" color={theme.color.yellow[300]} />
+            <ColorDemo name="400" color={theme.color.yellow[400]} />
+            <ColorDemo name="500" color={theme.color.yellow[500]} />
+            <ColorDemo name="600" color={theme.color.yellow[600]} />
+            <ColorDemo name="700" color={theme.color.yellow[700]} />
+          </Stack>
+          <Stack gapSize={0.5}>
+            <Heading variant="heading-3">Gray</Heading>
+            <ColorDemo name="100" color={theme.color.gray[100]} />
+            <ColorDemo name="200" color={theme.color.gray[200]} />
+            <ColorDemo name="300" color={theme.color.gray[300]} />
+            <ColorDemo name="400" color={theme.color.gray[400]} />
+            <ColorDemo name="500" color={theme.color.gray[500]} />
+            <ColorDemo name="600" color={theme.color.gray[600]} />
+          </Stack>
+          <Stack gapSize={0.5}>
+            <Heading variant="heading-3">White</Heading>
+            <ColorDemo name="White" color={theme.color.white} />
+          </Stack>
+        </ComponentsSection>
+      </Stack>
+    ),
+    [theme]
+  )
+
+  return (
+    <div
+      style={{
+        height: "100%",
+        width: "100%",
+        padding: theme.spacing.md,
+      }}
+    >
+      {memoizedChildren}
+    </div>
+  )
+}
+ColorTokens.storyName = "Color Tokens"
+ColorTokens.parameters = {
+  chromatic: { disable: true },
```

Comment:
Hmm...I wonder if maybe we'd actually want to test the colour palette?

Reply:
Actually that might me a good move, just to make sure we don't grunk up the colors at any point. Though if we want to do this maybe we leave it as a story instead of a page

Reply:
Also hesitant to make this a doc since we don't have pre-written documentation on it

Reply:
Maybe we could keep it as a story but remove the canvas? I wonder if that's possible 🤔 But if it's too much trouble we can leave it for a separate PR

Reply:
actually I stand corrected, there [totally is](https://cultureamp.design/guidelines/color/) ❗ 🧠  maybe we just leave this one as a test for colors